conf/pacific/cephfs/tier_1_fs.yaml

```diff
@@ -0,0 +1,49 @@
+globals:
```

Comment:
As it is same as the Tier 0 configuration, can we avoid this file and reuse the Tier-0?

We could create a new file when there is a new requirement/change in the test suite requirement.